www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -405,6 +405,7 @@
                     <Required>Y</Required>
                     <Default>tlssni</Default>
                     <OptionValues>
+                        <ssh>SSH</ssh>
                         <httphost>Host (HTTP)</httphost>
                         <tlssni>SNI (TLS)</tlssni>
                         <nottlssni>not SNI (TLS)</nottlssni>
```

Comment:
this listing could benefit from an alphabetical cleanup, possibly switching protocol and annotation:
HTTP (Host)
SSH
TLS (SNI)
TLS (non-SNI)
it's probably faster for the reader

Reply:
I want it to be the same order as the Routing Precedence.

TLS (non-SNI) is actually wrong. It is an inversion of SNI, only matching an inverted list of hostnames. Maybe there's a better name here?

Reply:
Actually your idea makes the most sense so I implemented it. Thanks.